### ISSUE TITLE
Is any plan to release the label for the test dataset?

### ISSUE THREAD

**Issue by YanLiang1102** [THU-KEG/MAVEN-dataset#3]
Or the plan is to always submit to the leaderboard?

**Comment by YanLiang1102** [comment]
To make the evaluation process easy, can u also release along the test dataset for each sentence what is the sentence id, (which is required in the final submit,  but it does not come with each sentences now, only with candidates, we can infer from there, but which make the process bit akward.  some of the sentence in the document if it is not in the candidates in the document, we do not have access to its sentenceid). Thank you @Bakser

**Comment by Bakser** [comment]
We do not have the plan to release the test labels to avoid overfitting on test set. You can tune hyperparameters on the released dev set and submit the predictions to the CodaLab leaderboard.

**Comment by zero0kiriyu** [comment]
Hi, I am now trying to implement an incremental learning algorithm on this dataset. However, it is impossible to evaluate the performance on a certain part of the test set without labels. It would be very helpful if you can release the label of the test set.
